doc/cite.rst

```diff
@@ -3,7 +3,7 @@
 How to cite MNE
 ---------------
 
-If you use in your research the implementations provided by the MNE software you should cite:
+If you use, in your research, the implementations provided by the MNE software you should cite:
```

Comment:
Not native either, but I would write:
`If you use the implementations provided by the MNE software in your research, you should cite:`